CMakeLists.txt

```diff
@@ -75,9 +75,9 @@ find_package(Boost REQUIRED)
 
 pkg_check_modules(xournalpp_deps REQUIRED IMPORTED_TARGET GLOBAL
         "glib-2.0 >= 2.32.0"
-        "gtk+-3.0 >= 3.22.0"
+        "gtkmm-3.0 >= 3.24.0"
```

Comment:
Do not require a too recent version or we will have problems with non-up-to-date operating systems like Debian or OpenSUSE.

Reply:
In prinziple this is fine, because it will require a long time until we can release that version (milestone 2.0). Then we could backport it anyways.

Reply:
Not quite sure Debian will have 3.24 by then but we can risk it 😉

Reply:
What creates more headaches is the usage of `zug` but that train has already departed (pun intended)

Reply:
Haha fun fact I wrote my thesis over delay prediction in passenger train systems.

Reply:
@LittleHuba zug is header only as well and the plan is to use gtk4 as soon as its released.

Reply:
Ahh well we should download those headers autmatically then (for devs). If not even Arch had a package before I created one we can assume no other OS will have. So we should make the life of potential devs not harder than it is and remove that hurdle.